pandas/tests/generic/test_generic.py

```diff
@@ -581,6 +581,10 @@ def test_validate_bool_args(self):
             with pytest.raises(ValueError):
                 super(DataFrame, df).mask(cond=df.a > 2, inplace=value)
 
+    def test_valid_deprecated(self):
+        with tm.assert_produces_warning(FutureWarning):
```

Comment:
add the issue number here

not sure this is actually in the right place. these are fairly generic routines

Reply:
Ok, I've moved the test to ``tests/generic/test_series.py``.